framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -25,6 +25,18 @@
 #include "audio/audio_manager.h"
 
 namespace media {
+const char *player_state_names[] = {
```

Comment:
I think placing it in the same location as the enum define would reduce the programmer's mistake.
or how about switch syntax.

Reply:
If I moved `player_state_names[]` to `MediaPlayerImpl.h`
Below build error occurs.
```
/home/tinyara/github/fork/os/../build/output/libraries/libframework.a(InputDataSource.o):(.data._ZN5media18player_state_namesE+0x0): multiple definition of `media::player_state_names'
/home/tinyara/github/fork/os/../build/output/libraries/libframework.a(MediaPlayer.o):(.data._ZN5media18player_state_namesE+0x0): first defined here
/home/tinyara/github/fork/os/../build/output/libraries/libframework.a(MediaPlayerImpl.o):(.data._ZN5media18player_state_namesE+0x0): multiple definition of `media::player_state_names'
/home/tinyara/github/fork/os/../build/output/libraries/libframework.a(MediaPlayer.o):/home/tinyara/github/fork/framework/src/media/MediaPlayer.cpp:24: first defined here
/home/tinyara/github/fork/os/../build/output/libraries/libframework.a(PlayerWorker.o):(.data._ZN5media18player_state_namesE+0x0): multiple definition of `media::player_state_names'
/home/tinyara/github/fork/os/../build/output/libraries/libframework.a(MediaPlayer.o):/home/tinyara/github/fork/framework/src/media/MediaPlayer.cpp:24: first defined here
```

Reply:
'const char* const player_state_names'
Then the error should disappear.